specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json

```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure HealthcareApis events published to Azure Event Grid.",
+    "description": "Describes the schema of the Azure HealthcareApis events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "FhirResourceCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.HealthcareApis.FhirResourceCreated event.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "Type of HL7 FHIR resource.",
+          "type": "string"
+        },
+        "resourceFhirAccount": {
+          "description": "Domain name of FHIR account for this resource.",
+          "type": "string"
+        },
+        "resourceFhirId": {
+          "description": "Id of HL7 FHIR resource.",
+          "type": "string"
+        },
+        "resourceVersionId": {
+          "description": "VersionId of HL7 FHIR resource. It changes when the resource is created, updated, or deleted.",
+          "type": "integer",
+          "format": "int64"
+        }
```

Comment:
As for now, we should have these 4 as critical properties of the event payload. They represent exact same states. Can we use a $ref tag to reduce duplication and make it easier for maintenance?

Example: 
- https://github.com/Azure/azure-rest-api-specs/blob/679887ace44697c726aba8d2814ee415a5d25e6f/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L663

Ref: 
- https://swagger.io/docs/specification/using-ref/
- https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/